samples/openthread/cli/README.rst

```diff
@@ -79,7 +86,7 @@ The sample supports the following development kits for testing the network statu
 
 .. table-from-rows:: /includes/sample_board_rows.txt
    :header: heading
-   :rows: nrf5340dk_nrf5340_cpuapp, nrf52840dk_nrf52840, nrf52833dk_nrf52833, nrf21540dk_nrf52840
+   :rows: nrf5340dk_nrf5340_cpuapp, nrf52840dk_nrf52840, nrf52840dongle_nrf52840, nrf52833dk_nrf52833, nrf21540dk_nrf52840
```

Comment:
Should we add it here as well? https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/app_boards.html?highlight=nrf52840dongle_nrf52840#boards-included-in-sdk-nrf